kernel/src/thread/mod.rs

```diff
@@ -111,6 +115,16 @@ impl Thread {
         self.priority.store(new_priority, Ordering::Relaxed)
     }
 
+    /// Acquires the lock of cpu affinity.
+    pub fn cpu_affinity_guard(&self) -> SpinLockGuard<CpuSet, PreemptDisabled> {
```

Comment:
Only simple getter methods are named as nouns. Normal methods should begin with a verb. So `lock_cpu_affinity` is a better name.